.github/buildomat/jobs/clippy.sh

```diff
@@ -30,4 +30,4 @@ ptime -m bash ./tools/install_builder_prerequisites.sh -y
 banner clippy
 export CARGO_INCREMENTAL=0
 ptime -m cargo xtask clippy
-ptime -m cargo doc
+RUSTDOCFLAGS="-Dwarnings" ptime -m cargo doc --workspace --keep-going
```

Comment:
Do we actually want the `--keep-going` flag here? Wouldn't that increase CI times for these workflows?

(I agree that it would be convenient locally, but I figured we'd want the CI to return any failures ASAP)

Reply:
It would increase the amount of time until GitHub tells you the job failed, yeah.

I don't know whether it's better to have the red X show up faster or have the logs show as many errors as possible (not necessarily all errors).

Reply:
(If you don't have a preference either, I'll hedge towards not changing it and drop it from this PR.)

Reply:
I *think* i have a mild preference for the default, aka, the "stop early" option, just because anything that gets results back from CI faster seems like a strong default unless we're convinced otherwise.